v3.0/how-to/upgrade/from-previous-version.md

```diff
@@ -122,22 +147,26 @@ $ ansible-playbook local_prepare.yml
 
 > **Note:**
 >
-> To optimize operation and maintenance management over components of TiDB cluster, there are some adjustments to  `PD service` under `systemd` in TiDB 3.0. Compared to earlier versions, rolling update operations on TiDB cluster components are slightly different. Please make sure that the `process_supervision` parameter remains consistent before and after the upgrade.
+> To optimize operation and maintenance management over components of TiDB cluster, there are some adjustments to `PD service` under `systemd` in TiDB 3.0. If the current version is earlier than TiDB 3.0 versions, the rolling update of TiDB cluster components to the 3.0 versions is slightly different. Please make sure that the `process_supervision` parameter remains consistent before and after the upgrade.
 
 - If the default `systemd` parameter is used by the `process_supervision` variable,perform rolling update on the TiDB cluster using  `excessive_rolling_update.yml`.
 
-   ```
-  $ ansible-playbook excessive_rolling_update.yml
-   ```
+    ```shell
+    $ ansible-playbook excessive_rolling_update.yml
+    ```
+
+    > **Note:**
+    >
+    > If the current version is later than TiDB 3.0 versions, then you can still use the `rolling_update.yml` approach to rolling-update and rolling-restart the TiDB cluster.
```

Comment:
```suggestion
    > If the current version is TiDB 3.0.0 or is later than TiDB 3.0.0, you can still use the `rolling_update.yml` approach to rolling-update and rolling-restart the TiDB cluster.
```

Reply:
@superlzs0476 Is this what you mean in https://github.com/pingcap/docs-cn/pull/1677?